### PR TITLE
feat: allow GITHUB_BASE_URL as an env variable

### PR DESCRIPTION
**Pull request by armaandhull**
resolves https://github.com/pascalgn/size-label-action/issues/38

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -29,6 +29,9 @@ const globrexOptions = { extended: true, globstar: true };
 async function main() {
   debug("Running size-label-action...");
 
+  // Set the GITHUB_BASE_URL constant value, using a default value of https://api.github.com if not provided
+  const GITHUB_BASE_URL = process.env.GITHUB_BASE_URL || "https://api.github.com";
```
Comment (armaandhull) [inline dist/index.js:33]: declare the constant GITHUB_BASE_URL and use a default value

File: dist/index.js [hunk 2]
```diff
@@ -63,6 +66,7 @@ async function main() {
   const pull_number = eventData.pull_request.number;
 
   const octokit = new Octokit({
+    baseUrl: GITHUB_BASE_URL,
```
Comment (armaandhull) [inline dist/index.js:69]: pass GITHUB_BASE_URL to Octokit constructor

**Review by armaandhull — COMMENTED** [review]
everything else is whitespace change due to mixed line endings in the file

**Comment by armaandhull** [comment]
@pascalgn bumping this up

**Comment by pascalgn** [comment]
@armaandhull Thanks for reminding me! I've decided to merge #43 instead because it had fewer changes and used GITHUB_API_URL, which is also used in https://github.com/pascalgn/automerge-action. Thanks for creating this PR and I hope your issue is still fixed now!

**Pull request closed without merging**